wire-ios-sync-engine/Source/Use cases/RemoveUserClientUseCase.swift

```diff
@@ -96,12 +96,20 @@ class RemoveUserClientUseCase: RemoveUserClientUseCaseProtocol {
             case .invalidCredentials, .missingAuth, .badRequest:
                 throw RemoveUserClientError.invalidCredentials
 
+            case .unknown:
+                switch failureResponse.code {
+                case TooManyRequestsStatusCode:
+                    throw RemoveUserClientError.tooManyRequests
+                default:
+                    throw RemoveUserClientError.generic
```

Comment:
question: we're loosing the underlying error here, no? should we not keep it for debugging as associated value?